CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+## Contributing Guidelines
+When contributing to the KOTS documentation, please consider the following writing styles:
+
+- ### Avoid acronyms. If using one, make sure to define it on first use.
+
+- ### Favor writing in the [active voice](http://writing.wisc.edu/Handbook/CCS_activevoice.html).
+**Yes:**  The next section includes some basic information about our app release including the app name.
+
+**No:**  In the next section some basic information is included about our app release including the app name.
+
+- ### Favor writing in the [indicative mood](https://grammarist.com/grammar/english-moods/).
+**Yes:**  At runtime, Replicated will rewrite the image tag to pull from `registry.replicated.com`.
+
+**No:**  At runtime, Replicated would rewrite the image tag to pull from `registry.replicated.com`.
+
+- ### Avoid Second Person Pronouns
+**Yes:** A license value can be used to show and hide fields on the settings page.
+
+**No:** You can use your license to allow customer access to hidden settings.
+
+- ### Avoid First Person Pronouns
+**Yes:** The Go playground can be used to validate the regular expression.
+
+**No:** Here I have used the Go playground to validate the regular expression.
+
+- ### Do not reference relative time from when the document was written.
+**Yes:** KOTS 1.15.0 includes a new feature to...
+
+**No:** A new feature of KOTS is...
+
+- ### One sentence per line. It's easier to review and comment on. Markdown will render these properly.
+**Yes:** Do something.
+Do something else.
+
+**No:** Do something. Do something else.
+
+- ### Use relevant, descriptive anchor text in links.
+**Yes:** ```More information is available in the <a href="/link">release notes</a>.```
+
+**No:**  ```For more information, read the release notes <a href="/link">here</a>.```
+
+- ### Only use Markdown backticks on code or command literals.
+The backtick symbol shouldn't be used for emphasis.
+
+- ### Code snippets in bash should not include sudo.
+**Yes:**  ```docker logs -f replicated```
+
+**No:** ```sudo docker logs -f replicated```
```

Comment:
I'd vote for this one to be removed. I actually prefer the latter, since it communicates that higher privileges is required for the command and that it has the highest change of working as-is. For example, we shouldn't remove the sudo from `curl ... | sudo bash` 

Or maybe we can revise with `don't use stuff in example commands if it's not needed`, if that really needs to be said.

Reply:
@marccampbell what do you think? I'd like to get this merged in soon. 

@austinchambers while your argument isn't totally off and I'm interested to hear why we originally settled on the `no sudo in commands` convention, I think we've had this convention in place on all our docs sites for years now, and changing it should take place in a separate proposal.